.github

```diff
@@ -0,0 +1,48 @@
+name: Linters
+
```

Comment:
- [ ] Please your linters are not working as I noticed and as the last reviewer told you she described the problem " Kindly setup [GitHub actions ](https://github.com/microverseinc/linters-config/tree/master/html-css#set-up-github-actions)by adding the .github/workflows folder and add a copy of [.github/workflows/linters.yml](https://github.com/microverseinc/linters-config/blob/master/html-css/.github/workflows/linters.yml), you can also add [.stylelintrc.json](https://github.com/microverseinc/linters-config/blob/master/html-css/.stylelintrc.json) and [.hintrc](https://github.com/microverseinc/linters-config/blob/master/html-css/.hintrc) to your root project so you can set up linters locally, also don't forget to add node_modules to .gitignore so you don't add it to git 🤝"

there should be 3 checks here for web hint ,silent and Lighthouse